zk/stages/stage_sequence_execute.go

```diff
@@ -190,6 +189,33 @@ func SpawnSequencingStage(
 		return err
 	}
 
+	var forkId uint64 = 0
+
+	if executionAt == 0 {
+		// capture the initial sequencer fork id for the first batch
+		forkId = cfg.zk.SequencerInitialForkId
+		for fId := uint64(chain.ForkID5Dragonfruit); fId <= forkId; fId++ {
+			if err := hermezDb.WriteForkId(1, fId); err != nil {
```

Comment:
This should be outside the loop as it's the fork id for just this block

Reply:
The goal of this loop is to enable all forkids from 5 up to cfg.zk.SequencerInitialForkId. Otherwise function like "IsForkID7Etrog" in chain-config return false while they must return true.

Reply:
I agree, but that is driven by the `WriteForkIdBlockOnce`.  The `WriteForkId` is a per block thing, and a block can only belong to one fork

Reply:
I see, thanks. The code is updated.